packages/amplify-provider-awscloudformation/src/utility-functions.js

```diff
@@ -60,6 +62,13 @@ module.exports = {
 
     return transformGraphQLSchema(context, optionsWithUpdateHandler);
   },
+
+  buildOverrides: async (context, options) => {
```

Comment:
Since this is a generic utility file, please add a comment as to whats the function of this file and what it generates.